### PR TITLE
hwdb : add support for Archos 101 Cesium to 60-sensor.hwdb

### PR DESCRIPTION
**Pull request by tomlohave**

### CODE REVIEW

**Review by yuwata — CHANGES_REQUESTED** [review]
Please merge the new entry with the first entry for pnARCHOS80Cesium

File: hwdb.d/60-sensor.hwdb [hunk 1]
```diff
@@ -124,14 +124,12 @@ sensor:modalias:acpi:MXC6655*:dmi:*:svnAquarius*:pnNS483:*
 # Archos
 #########################################
 sensor:modalias:acpi:SMO8500*:dmi:*:svnARCHOS:pnARCHOS80Cesium:*
+sensor:modalias:acpi:KIOX000A*:dmi:*:svnARCHOS:pnARCHOS101CESIUM:*
```
Comment (yuwata) [inline hwdb.d/60-sensor.hwdb:130]: Could you add a brief comment here?
`# Archos 101 Cesium`
(Also for the first entry if possible, that I guess Archos 80 Cesium ??)

Comment (tomlohave) [reply hwdb.d/60-sensor.hwdb:130]: Sure I can !

**Review by yuwata — APPROVED** [review]
Added further update for the comment. LGTM now.

**Pull request merged**